UnityProject/Assets/Scripts/NPC/AI/MobController.cs

```diff
@@ -3,30 +3,56 @@
 using System.Collections.Generic;
 using System.Linq;
 using UnityEngine;
+using UnityEngine.SceneManagement;
 
 namespace Systems.MobAIs
 {
 	public class MobController : MonoBehaviour
 	{
+
+		public static int CurrentMobs = 0;
+
+		public bool Active = false;
+
 		public List<MobObjective> MobObjectives = new List<MobObjective>();
 
+
+		void OnRoundRestart(Scene oldScene, Scene newScene)
+		{
+			CurrentMobs = 0;
+		}
+
 		public void Awake()
 		{
+
 			MobObjectives = this.GetComponents<MobObjective>().ToList();
 		}
 
 		private void OnEnable()
 		{
 			if (CustomNetworkManager.IsServer == false) return;
-		
+			SceneManager.activeSceneChanged += OnRoundRestart;
+			if (CurrentMobs > 20)
+			{
+				return;
```

Comment:
Could we send this to admin chat? Just to keep them informed

Reply:
ok cool will do